src/webview/static/base.css

```diff
@@ -690,3 +690,49 @@ h1, h2, h3, h4, h5, h6 {
 .spoiler-block .spoiler-arrow.spoiler-button-open::after {
   transform: rotate(90deg) translate(10px, 0);
 }
+
+/* Poll styling */
+
+.poll-widget {
+  border: hsl(0, 0%, 50%) 1px solid;
+  padding: 2px 8px 2px 10px;
```

Comment:
Why more left padding than right padding? Is it something like what Greg said at https://github.com/zulip/zulip-mobile/pull/4750#discussion_r639285725, about one of Material Design's decisions:

> Speculating, I think the idea in giving the text more left padding than right padding may be that because it's justified left, there's naturally some space between the actual text and the right edge of its box -- and so the text will look more centered if the box is actually somewhat right of center.

I notice that it makes the dividing line between the question and the options a bit off-center, horizontally:

![image](https://user-images.githubusercontent.com/22248748/120570902-13ae8c00-c3e7-11eb-876a-727e6e2d7c9d.png)

This might be fine; is it deliberate?

Reply:
This was copied from the `spoiler-block` css, I'm not sure why it's like that.